setup.py

```diff
@@ -19,13 +19,24 @@
 import os
 from distutils.core import setup, Extension
 from distutils.command.build import build
+import re
 import subprocess
 import versioneer
 
+EXCLUDE = [
+    r'void glp_vprintf(.+);',
+    r'^.+rmfgen.+\n*.+;',
+    r'^.+netgen.+\n*.+;',
+    r'^.+gridgen.+\n*.+;',
+]
+
+
 def copy_glpk_header():
     if os.path.isfile('glpk.h'):
         print('glpk.h found in source directory')
         glpk_header_path = os.path.join('./', 'glpk.h')
+    elif "GLPK_HEADER_PATH" in os.environ:
```

Comment:
This should make it easier to build on conda-forge in the future :crossed_fingers: